src/utils/originRegExpSerde.ts

```diff
@@ -0,0 +1,41 @@
+import type { CorsOptions } from '../module';
```

Comment:
Could we have some basic test cases added to the test suite that exemplify the use case - in which case it should work and should not work